### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by ra0x3**

### CODE REVIEW

File: docs/src/components/database/types.md [hunk 1]
```diff
@@ -14,18 +14,21 @@ Below is a mapping of GraphQL schema types to their Postgres equivalents, refere
 | str[32] | Salt | varchar(64) |
 | u32 | UInt4 | integer |
 | u64 | UInt8 | bigint |
-| u64 | Timestamp | timestamp |
+| i64 | Timestamp | timestamp |
 | str[] | Blob | bytes |
 | str[32] | MessageId | varchar(64) |
+| bool | Boolean | bool |
+|  | Json | json |
```
Comment (deekerno) [inline docs/src/components/database/types.md:21]: Are these meant to be empty?

File: docs/src/application-dependencies/index.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Application dependencies
```
Comment (deekerno) [inline docs/src/application-dependencies/index.md:1]: Should we add the corresponding subsections on this page?

File: docs/src/getting-started/fuel-indexer-project.md [hunk 3]
```diff
@@ -1,11 +1,16 @@
 # A Fuel Indexer Project
 
-The Fuel indexer service can currently be used in two different ways:
+The Fuel indexer project can currently be used in two different ways:
```
Comment (deekerno) [inline docs/src/getting-started/fuel-indexer-project.md:3]: s/two/three

**Review by deekerno — CHANGES_REQUESTED** [review]
Looking good overall; left a few comments.

File: docs/src/components/database/types.md (hunk 1 above)
Comment (ra0x3) [reply docs/src/components/database/types.md:21]: Yes there's no sway equivalent

**Review by deekerno — APPROVED** [review]

**Pull request merged**